TLM/TLM/UI/SubTools/LaneConnectorTool.cs

```diff
@@ -805,15 +805,20 @@ private class LaneEnd {
         }
 
         private static int CountLanes(ushort segmentId, ushort nodeId, bool toward) {
-            ExtSegmentManager extSegmentManager = ExtSegmentManager.Instance;
-            return extSegmentManager.GetSortedLanes(
-                                segmentId,
-                                ref segmentId.ToSegment(),
-                                ExtSegmentManager.Instance.IsStartNode(segmentId, nodeId) ^ (!toward),
```

Comment:
I've no idea how this ever worked given that `ExtSegmentManager.Instance.IsStartNode()` can return `null`.

Reply:
it never does in practice :). unless the game is really broken!